streams/byte_slice_stream.ts

```diff
@@ -48,7 +48,9 @@ export class ByteSliceStream extends TransformStream<Uint8Array, Uint8Array> {
     super({
       start: () => {
         if (start < 0) {
-          throw new RangeError("`start` must be greater than 0");
+          throw new RangeError(
+            "Invalid argument: `start` must not be negative",
```

Comment:
```suggestion
            "Invalid argument: `start` is negative",
```

Reply:
```suggestion
            `Invalid argument, 'start' must >= 0: current value ${start}` ,
```

Reply:
Updated to `'start' must be >= 0: received ${start}`

Reply:
Settled on `Cannot construct ByteSliceStream as start must be >= 0: received ${start}`